.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
add the `.gitignore` file before your first commit. otherwise it does not work. That is why all your node_modules are on github